include/linux/sysfs_helpers.h

```diff
@@ -0,0 +1,53 @@
+/*  Sysfs helper functions
+ *
+ *  Author: Andrei F. <luxneb@gmail.com>
+ *  Derived from function implementation from Gokhan Moral
+ *  
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ */
+
+static inline int read_into(int *container, int size, const char *buf, size_t count)
```

Comment:
What is this function for? I can't any place where a function calls read_into?

Reply:
This is for overclocking, if you consider it in the future.